src/Post/Features/SettingsPage.php

```diff
@@ -15,76 +17,23 @@
  */
 trait SettingsPage {
 
-	private $_settings_title;
+	private $_page;
 
 	protected function _init_SettingsPage() {
 		if ( !empty( $this->settings_form ) ) {
-			add_action( 'admin_menu', [ $this, '_register_settings_page' ] );
-			add_action( 'admin_init', [ $this, '_register_setting' ] );
-			$this->_settings_title = !empty( $this->settings_title ) ? $this->settings_title : 'Settings';
+			$this->_page = new Page( OPTIONS_PREFIX . $this->get_post_type(), [
```

Comment:
For clarity, maybe use the full reference to the `OomphInc\Bases\Admin\SettingsPage` class instead, especially since it's only referenced once.